core/src/main/scala/org/apache/spark/api/java/JavaPairRDD.scala

```diff
@@ -223,7 +223,9 @@ class JavaPairRDD[K, V](val rdd: RDD[(K, V)])
   /**
    * Generic function to combine the elements for each key using a custom set of aggregation
    * functions. Turns a JavaPairRDD[(K, V)] into a result of type JavaPairRDD[(K, C)], for a
-   * "combined type" C. Note that V and C can be different -- for example, one might group an
+   * "combined type" C.
+   *
+   * @note V and C can be different -- for example, one might group an
```

Comment:
I was skimming the changes again and noticed this. This causes this note to render pretty separately from this text, but it seems to belong with the sentence above. 

Does it cause the bullet points below to be in the note? because they also seem unrelated to this 'note'.

Could be OK but wasn't sure whether this was _really_ a stand-alone note as much as a usage of the word "note"